ftp/HandelGF/BWV351/LaRej/LaRej.ly

```diff
@@ -81,21 +82,21 @@ orgueUP = \relative c'{
     <cis e>4. <a d>16 <d fis> <cis e>8.~ <d e>16 <e>8 <cis a'>      | % 12
     <fis a>~ <fis b>16~ <fis cis'>~ <fis d'>8 <g b>16~ <e b'> <fis a>4. <e g>16~ <g b>      | % 13
     <fis a>4. <e g>16~ <g b> <fis a>4. <fis a>8      | % 14
-    <g b> <g b>8. <fis a>16 <e g>~ <g b> <fis a>8 <fis a>8.~ <e g>16 <d fis>~ <fis a>      | % 15
-    <e g>8 <e g>8.~ <d fis>16 <cis e>~ <e g> <d fis>8~ <a fis'>~ <d fis>16~ <d e> <d>~ <d fis>~      | % 16
-    <d e>8 <cis e>16~ <b e> <cis e>~ <d fis> <e g>~ <cis e> <a fis'>8~ <a d> <a d> <d e>16 <d fis>      | % 17
-    <e g>8 <cis e> <cis e> <d fis>16~ <e g> <fis a>8 <d fis> <d fis> <e g>16~ <fis a>      | % 18
+    <g b> <g b>8. <fis a>16 <e g>~ <g b> <fis a>8 <fis a>8. <e g>16 <d fis>~ <fis a>      | % 15
+    <e g>8 <e g>8. <d fis>16 <cis e>~ <e g> <d fis>8~ <a fis'>~ <d fis>16~ <d e> <d>~ <d fis>~      | % 16
+    <d e>8 <cis e>16~ <b e> <cis e> <d fis> <e g>~ <cis e> <a fis'>8~ <a d> <a d> <d e>16 <d fis>      | % 17
+    <e g>8 <cis e> <cis e> <d fis>16 <e g> <fis a>8 <d fis> <d fis> <e g>16 <fis a>      | % 18
     <g b>8. <e g>16 <fis a>8. <d fis>16 <cis g'>8 <d fis>16~ <d e> <e a>8 <cis e g>      | % 19
-    <d fis>~ <b fis'>16~ <b e> <cis e>8.~ <a d>16 <fis a d>4.~ r8 \bar "|."  % 20
+    <d fis>~ <b fis'>16~ <b e> <cis e>8. <a d>16 <fis a d>4. r8 \bar "|."  % 20
```

Comment:
perhaps the ~ of fis should be moved inside the chord?  (wherever the tie is possible)

    <d fis~> <b fis'>